src/parser/tests/expression/TernaryExpression.spec.ts

```diff
@@ -274,30 +274,30 @@ describe('ternary expressions', () => {
         });
 
         it('simple consequents', () => {
-            testTranspile(
-                `a = user = invalid ? "no user" : "logged in"`,
-                `a = bslib_ternary(user = invalid, "no user", "logged in")`
-            );
-
-            testTranspile(
-                `a = user = invalid ? 1 : "logged in"`,
-                `a = bslib_ternary(user = invalid, 1, "logged in")`
-            );
-
-            testTranspile(
-                `a = user = invalid ? 1.2 : "logged in"`,
-                `a = bslib_ternary(user = invalid, 1.2, "logged in")`
-            );
+            // testTranspile(
```

Comment:
Could be a good idea to mention why these are left here. Either they are not going to be used and were forgotten, or will be included someday in the test, in which case a TODO comment is appreciated :+1:

Reply:
Oops! Yeah, those should not have been commented out. Re-enabled, and tests still pass!